### ISSUE TITLE
Sometimes enemies do not take damage from burning

### ISSUE THREAD

**Issue by squid-lee** [dbjorge/jorbs-spire-mod#170]
I've seen this twice I think in about 3 hours, and I can't repro it consistently.
I think it might be to do with flame ward?

The debuff icon is there, but it doesn't tick down or deal damage.

**Comment by dbjorge** [comment]
Burning intentionally has a "final turn" after ticking down from 1 where it won't do any more damage but will still do the 50% healing effect; in that state, you'll see the icon, but it won't have a number on it.

Does that sound like what you're remembering, or are you thinking of something else where it definitely had a number on the icon but still didn't tick properly?

**Comment by squid-lee** [comment]
Definitely the latter - I had a 'wound slaver' with 28 burn who just wasn't taking damage or having it tick down

**Comment by dbjorge** [comment]
If it wasn't ticking down, that's definitely a bug. Since you mentioned Flame Ward, is it possible that it was ticking down, but the damage was being absorbed by block? it's intended for the damage to be blockable, though that isn't clear from the current text (updating it from "lose X hp" to "take X damage" to clarify this).

I'm not able to reproduce a case where it doesn't tick down at all and don't see any reason that would happen from looking at the code a bit; if you see that again it's definitely a bug we want to fix, but I'm afraid we're going to need more specific steps to reproduce it before we can fix it.

**Comment by dbjorge** [comment]
4fe373a covers improving the wording from "lose hp" to "take damage"

**Comment by squid-lee** [comment]
Happened again:
3xslavers elite Act 2, I play flame ward+.
The first slaver takes all 3x 10 burn damage, and dies.
The other two slavers get the debuff icon, but take no damage from the debuff, or see the debuff tick down

**Comment by wang429** [comment]
I think some weird stuff was noted with flame ward and burning in jorbs's stream

**Comment by dbjorge** [comment]
Was able to repro it with flame ward + slavers, thanks for providing more info! Should be fixed for next release.
